Formula/ntfs-3g.rb

```diff
@@ -38,25 +35,23 @@ class Ntfs3g < Formula
   end
 
   on_linux do
-    depends_on "libfuse"
+    depends_on "libfuse@2"
   end
 
   def install
-    ENV.append "LDFLAGS", "-lintl"
+    on_macos { ENV.append "LDFLAGS", "-lintl" }
```

Comment:
```suggestion
    ENV.append "LDFLAGS", "-lintl" if OS.mac?
```
We don't use `on_macos` in `install` blocks anymore.

Reply:
I saw this, but wanted to wait until we determined Homebrew's stance on maintaining macOS-specific installation steps for macOS-disabled FUSE formulae.

Reply:
At this point, might as well just keep macOS-specific logic rather than blocking PR on response. So, added suggestion.